<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="quad1" default="SQ01s"/>
    <arg name="quad2" default="SQ02s"/>

    <node ns="$(arg quad1)" name="obstacle" pkg="rmader" type="obstacle.py" output="screen" clear_params="true">
        <rosparam file="$(find rmader)/param/obstacle1.yaml" command="load"/>
        <param name="obs_id" value="4001" />
    </node>

    <node ns="$(arg quad1)" name="rmader_obs_commands" pkg="rmader" type="rmader_obs_commands.py" output="screen"/>

    <node ns="$(arg quad1)" pkg="rqt_gui" type="rqt_gui" name="obs_rqt_gui" args="--perspective-file $(find rmader)/rqt_cfgs/obs_demo_interface.perspective"/>

    <node ns="$(arg quad1)" pkg="behavior_selector" type="behavior_selector_node.py" name="obs_behavior_selector" output="screen" />

    <node ns="$(arg quad2)" name="obstacle" pkg="rmader" type="obstacle.py" output="screen" clear_params="true">
        <rosparam file="$(find rmader)/param/obstacle2.yaml" command="load"/>
        <param name="obs_id" value="4002" />
    </node>

    <node ns="$(arg quad2)" name="rmader_obs_commands" pkg="rmader" type="rmader_obs_commands.py" output="screen"/>

    <node ns="$(arg quad2)" pkg="rqt_gui" type="rqt_gui" name="obs_rqt_gui" args="--perspective-file $(find rmader)/rqt_cfgs/obs_demo_interface.perspective"/>

    <node ns="$(arg quad2)" pkg="behavior_selector" type="behavior_selector_node.py" name="obs_behavior_selector" output="screen" />

</launch>